source/Octopus.Tentacle/Scripts/Kubernetes/RunningKubernetesJobScript.cs

```diff
@@ -0,0 +1,323 @@
+        }
+
+
+
+        async Task CreateJob(IScriptLogWriter writer, CancellationToken cancellationToken)
+        {
+            //write the bootstrap runner script to the workspace
+            workspace.WriteFile("bootstrapRunner.sh", await BootstrapRunnerScript);
+
+            var scriptName = Path.GetFileName(workspace.BootstrapScriptFilePath);
+
+            var jobName = jobService.BuildJobName(scriptTicket);
+
+            //Deserialize the volume configuration from the environment configuration
+            var volumes = KubernetesYaml.Deserialize<List<V1Volume>>(KubernetesConfig.JobVolumeYaml);
+
+            var job = new V1Job
+            {
+                ApiVersion = "batch/v1",
+                Kind = "Job",
+                Metadata = new V1ObjectMeta
+                {
+                    Name = jobName,
+                    NamespaceProperty = KubernetesConfig.Namespace,
+                    Labels = new Dictionary<string, string>
+                    {
+                        ["octopus.com/serverTaskId"] = taskId,
+                        ["octopus.com/scriptTicketId"] = scriptTicket.TaskId
+                    }
+                },
+                Spec = new V1JobSpec
+                {
+                    Template = new V1PodTemplateSpec
+                    {
+                        Spec = new V1PodSpec
+                        {
+                            Containers = new List<V1Container>
+                            {
+                                new()
+                                {
+                                    Name = jobName,
+                                    Image = await GetDefaultContainer(),
+                                    Command = new List<string> { "bash" },
+                                    Args = new List<string>
+                                    {
+                                        $"/data/tentacle-home/{instanceName}/Work/{scriptTicket.TaskId}/bootstrapRunner.sh",
+                                        $"/data/tentacle-home/{instanceName}/Work/{scriptTicket.TaskId}",
+                                        $"/data/tentacle-home/{instanceName}/Work/{scriptTicket.TaskId}/{scriptName}"
+                                    }.Concat((workspace.ScriptArguments ?? Array.Empty<string>())
+                                        .SelectMany(arg => new[]
+                                        {
+                                            $"'{arg}'"
+                                        })).ToList(),
+                                    VolumeMounts = new List<V1VolumeMount>
+                                    {
+                                        new("/data/tentacle-home", "tentacle-home"),
+                                    },
+                                    Env = new List<V1EnvVar>
+                                    {
+                                        new(EnvironmentVariables.TentacleHome, $"/data/tentacle-home/{instanceName}"),
```

Comment:
Some of those other environment variables sound like they could be relevant for Calamari. `TentacleJournal` or `TentacleInstanceName`?

Reply:
hrm, good call. I'll investigate. I added these ones because they were needed for the health-check

Reply:
Yep, I missed `TentacleJournal` and `TentacleInstanceName`. Added